cypress/integration/visual-testing/homepage-test.spec.js

```diff
@@ -0,0 +1,41 @@
+import { baseURL, eyesOpen } from '../../utils';
+describe('Visual  Regression Tests', () => {
+  beforeEach(() => {
+    cy.visit(baseURL);
+    eyesOpen('Home Page');
+  });
+  afterEach(() => {
+    cy.eyesClose();
+  });
+
+  it('should display sign up options', () => {
+    cy.get('#navigation-butt').click();
+    cy.get('#navigation-butt').should('be.visible');
+  });
+
+  it('should redirect to code of conduct', () => {
+    const elem = cy.get('[data-cy-href=codeofconduct]');
+    elem.contains('Code of Conduct');
+    return elem.invoke('attr', 'href').then(href => {
+      cy.visit(`${baseURL}${href}`);
+      cy.wait(2000);
+      cy.get('h1')
+        .should('be.visible')
+        .contains('Code of Conduct');
+    });
+  });
+
+  it('should redirect to about us', () => {
+    cy.get('#loggedoutmenu.menu.logged-out').invoke('show');
```

Comment:
Consider using the `data-cy` attribute instead. Relying on selectors that are CSS classes or IDs may change over time causing an e2e test fail.